src/Controller/AppController.php

```diff
@@ -43,6 +43,10 @@ public function initialize()
 
         $this->loadComponent('RequestHandler');
         $this->loadComponent('Flash');
+        //Recommended security enhancements for your application
```

Comment:
For better readability, may be add a white line.

Please add a space affter ``//`` for comments.
For the commented out code, it's OK.